3scaleAdapter/README.md

```diff
@@ -33,7 +34,23 @@ To deploy the adapter to a Kubernetes/OpenShift cluster, within the `istio-syste
 oc create -f deploy/
 ```
 
-## Customise and create the adapter configuration
+## Configuring the adapter
+
+The runtime behaviour of the adapter can be modified by editing the deployment and setting or
+configuring the following environment variables:
+
+| Variable                         | Description                                                                                        | Default |
+|----------------------------------|----------------------------------------------------------------------------------------------------|---------|
+| THREESCALE_LISTEN_ADDR           | Sets the listen address for the gRPC server                                                        | 0       |
+| THREESCALE_LOG_LEVEL             | Sets the minimum log output level. Accepted values are one of `debug`,`info`,`warn`,`error`,`none` | info    |
+| THREESCALE_LOG_JSON              | Controls whether the log is formatted as JSON                                                      | true    |
+| THREESCALE_REPORT_METRICS        | Controls whether 3scale system and backend metrics are collected and reported to Prometheus        | true    |
+| THREESCALE_METRICS_PORT          | Sets the port which 3scale `/metrics` endpoint can be scrapped from                                | 8080    |
+| THREESCALE_CACHE_TTL_SECONDS     | Time period, in seconds, to wait before purging expired items from the cache                       | 300     |
+| THREESCALE_CACHE_REFRESH_SECONDS | Time period before expiry, when cache elements are attempted to be refreshed                       | 180     |
```

Comment:
perhaps explain a bit what combinations of TTL and refresh durations make (or don't make) sense.